### PR TITLE
Update master branch to main

### PR DESCRIPTION
**Pull request by TheJuanAndOnly99**
https://github.com/finos/FDC3/issues/579

Replaced `master` with `main` where appropriate.

### CODE REVIEW

**Review by robmoffat — APPROVED** [review]
amazing effort!

**Comment by maoo** [comment]
> replaced `master` with `main` where appropriate.

@TheJuanAndOnly99 - solid work here! I've just dropped a few comments, but since you're off today, I'll just make the changes on my end and drop a comment when I think this is ready to be merged.

**Comment by maoo** [comment]
@robmoffat @kriswest - this PR is now ready to be reviewed/merged!

**Comment by robmoffat** [comment]
Hi @maoo,

Once (if) Kris has approved, we should book some time and do this on a call - that way we can do some thorough testing and deal with any unknowns that pop up.

@kriswest - please review.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [![Latest Standard](https://img.shields.io/badge/release-2.0-blue)](https://github.com/finos/fdc3/releases/v2.0)
 [![npm](https://img.shields.io/npm/v/@finos/fdc3)](https://www.npmjs.com/package/@finos/fdc3)
-[![FINOS - Released](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-released.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Released)
+[![FINOS - Released](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@main/images/badge-released.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Released)
```
Comment (bingenito) [inline README.md:7]: We sure about this one?

File: docs/fdc3-charter.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The group's activities do not include:
 
 - Versions of the FDC3 specification following 2.1 and subsequent draft specifications are licensed under the [Community Specification License 1.0](https://github.com/finos/FDC3/blob/4ce90d45ca8e0c4f8f2c5bd73f51304278783d87/LICENSE.md)
```
Comment (bingenito) [inline docs/fdc3-charter.md:37]: Should this point at head version of license?

File: src/app-directory/specification/appd.schema.json [hunk 3]
```diff
@@ -904,7 +904,7 @@
 		  }
 		},
 		"IntentV1": {
-		  "description": "(Deprecated v1 API version) An intent definition as defined by spec https://github.com/FDC3/Intents/blob/master/src/Intent.yaml",
+		  "description": "(Deprecated v1 API version) An intent definition as defined by spec https://github.com/FDC3/Intents/blob/main/src/Intent.yaml",
```
Comment (bingenito) [inline src/app-directory/specification/appd.schema.json:907]: These legacy links are all 404, should they just be removed?

File: website/versioned_docs/version-2.0/fdc3-charter.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ To be successful, the maintenance and evolution of this Standard needs to have a
 ## Licensing
 Version 1.0 of the FDC3 specification is licensed under the [FDC3 1.0 Final Specification License](https://github.com/finos/FDC3/blob/17892008c26a73ff1fd9f6e40ceb8c8bfd58c610/PATENTS-FDC3-1.0.md).
```
Comment (bingenito) [inline website/versioned_docs/version-2.0/fdc3-charter.md:41]: Should this link to patents file on head?

File: README.md (hunk 1 above)
Comment (bingenito) [reply README.md:7]: Confirmed that it is main and good, but note for clarity that it is not directly related to this branch name change on this project.

File: website/versioned_docs/version-2.0/fdc3-charter.md (hunk 4 above)
Comment (robmoffat) [reply website/versioned_docs/version-2.0/fdc3-charter.md:41]: no, this is for the 2.0 version of FDC3, which is using the old license.

File: docs/fdc3-charter.md (hunk 2 above)
Comment (finos-admin) [reply docs/fdc3-charter.md:37]: We prefer using hashes, as it's not mutable; even though we don't plan to change the license in FDC3, we thought it was a good practice to follow.

**Pull request merged**